data/json/items/tool_armor.json

```diff
@@ -3329,5 +3329,109 @@
     ],
     "flags": [ "FANCY", "VARSIZE", "SUN_GLASSES" ],
     "material_thickness": 0.5
+  },
+  {
+    "id": "utility_exoskeleton_off",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "battery powered utility exoskeleton" },
+    "//": "This is based on the Sarcos Guardian, a production model with independent battery power, all weather operation, and capable of lifting up to 200 lbs. It uses three 12 lb batteries that provide 8 hours of operation. The battery life in-game is significantly less to utilize medium storage batteries, otherwise this would require the unit to be active to reload itself.",
```

Comment:
```suggestion
    "//": "This is based on the Sarcos Guardian, a production model with independent battery power, all weather operation, and capable of lifting up to 200 lbs. It uses three 12 lb batteries that provide 8 hours of operation. The battery life in-game is significantly less in order to utilize medium storage batteries, otherwise this would require the unit to be active to reload itself. This can also be changed out to use other battery types.",
```